Sources/Sentry/SentryQueueableRequestManager.m

```diff
@@ -27,6 +27,20 @@ - (instancetype)initWithSession:(NSURLSession *)session
 
 - (BOOL)isReady
 {
+#if TEST || TESTCI
```

Comment:
`m`: I would prefer a different type of implementation of the protocol `SentryRequestManager` that implements the behavior we need for testing. I think this class shouldn't know about this test functionality.